src/SmartComponents/HistoricalProfilesPopover/__tests__/__snapshots__/HistoricalProfilesPopover.tests.js.snap

```diff
@@ -532,7 +532,7 @@ exports[`ConnectedHistoricalProfilesPopover should render correctly 1`] = `
           >
             <span
               className="hsp-icon-padding historical-system-profile-popover"
-              data-ouia-component-id="hsp-popover-toggle"
+              data-ouia-component-id="hsp-popover-toggle-undefined"
```

Comment:
I'm not sure what you want to do here. There are instances where the id is undefined. Do you want to keep `undefined` here? Or do you want to keep it as `hsp-popover-toggle`?

Reply:
I didn't know that. What cases can cause an HSP id to be undefined?

Reply:
I think we can leave it as undefined for now